lib/cerebrum/helper.rb

```diff
@@ -1,38 +1,27 @@
 module Helpers
-  extend self
-
   # [{a: 1}, {b: 6, c: 7}] -> {a: 0, b: 1, c: 2}
-  def build_lookup(hashes)
-    hash = {}
-    hashes.map do |k|
-      k.each do |v|
-        hash[v[0]] = v[1]
-      end
-    end
-    lookup_from_hash(hash)
+  def features_to_vector_index_lookup_table(features)
+    flattened_feature_keys = features.inject(:merge)
+    reindex_hash_values(flattened_feature_keys)
```

Comment:
@arhamahmed changed this to make it slightly more readable, `:merge` should make more sense than `:update`.